src/routes/Voyages.svelte

```diff
@@ -1,32 +1,31 @@
 <script>
-  import Profile from "../profile/Profile.svelte";
   import { Collection, User } from "sveltefire";
   import { db } from "../firebase";
   import { link } from "svelte-spa-router";
 
-  export const addVoyage = () => {
-    db.collection("voyages").doc().set({ name: "YeeHaw Voyage" });
+  const addVoyage = () => {
+    db.collection("voyages").doc().set({ name: voyageName });
   };
+  let voyageName = "Voyage Name";
```

Comment:
You should have voyagename start out as an empty string and add a placeholder.
RIght now the user has to delete the text "Voyage name" before inputting the name